stable/artifactory-ha/README.md

```diff
@@ -495,6 +496,31 @@ Include the secret's name, along with the desired hostnames, in the Artifactory
           - artifactory.domain.com
 ```
 
+### Ingress annotations
```

Comment:
@zendril We should specify that these annotations are only to use Artifactory as Docker Registry with   Repository Path as a reverse proxy method.

Reply:
Fair point. I'll update.

Reply:
@jainishshah17 Done and pushed.